FinniversKit/Sources/Assets/Icons.xcassets/MotorTransaction/carFront.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
+  "images" : [
+    {
+      "filename" : "Size=24.svg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "Size=32.svg",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
```

Comment:
I'm not completely sure, but it doesn't seem like this image _(added to FinniversKit and not the demo)_ is used anywhere so it can probably be removed :)